applications/matter_weather_station/configuration/thingy53_nrf5340_cpuapp/prj.conf

```diff
@@ -97,13 +91,13 @@ CONFIG_PWM=y
 
 # Configure UART logging and shell
 CONFIG_LOG=y
+CONFIG_LOG2_MODE_DEFERRED=y
 CONFIG_LOG_BACKEND_UART=y
 CONFIG_LOG_PRINTK=y
 CONFIG_LOG_PROCESS_THREAD_STACK_SIZE=1024
 CONFIG_LOG_PROCESS_THREAD_STARTUP_DELAY_MS=4000
-CONFIG_LOG_STRDUP_BUF_COUNT=48
-CONFIG_LOG_STRDUP_MAX_STRING=100
-CONFIG_LOG_BUFFER_SIZE=2048
+CONFIG_LOG_BUFFER_SIZE=4096
```

Comment:
I wonder if your fix of "unable to allocate" errors would be necessary to add here? I guess we should add this config `CONFIG_LOG_STRDUP_BUF_COUNT=100` in that case.

Reply:
`LOG2_MODE_DEFERRED` + log buffer size increased should be enough.